relay-server/src/services/project_cache.rs

```diff
@@ -683,8 +663,10 @@ impl ProjectCacheBroker {
             no_cache,
         );
 
-        if !state.invalid() {
-            self.dequeue(project_key);
+        // Schedule unspool if nothing is running at the moment.
```

Comment:
Should we check here too whether there are enough buffer guards available?

Reply:
nope, the check is done in `handle_periodic_unspool` now.